dist/api_data.py

```diff
@@ -478,6 +478,10 @@ def __cmp__(self, other):
                 vary depending on the current eviction load''',
                 min=1, max=20),
             ]),
+    Config('readonly', 'false', r'''
+        open connection in read-only mode.  The database must exist.  All
+        methods that may modify a database are disabled.''',
```

Comment:
Yes.  I do still need to do that.  Michael and I had discussed that it needs its own page in the Ref Guide.  Good suggestion to add the ref there too.